chains/osmosis/scripts/localnet.mk

```diff
@@ -0,0 +1,126 @@
+###############################################################################
+###                                Localnet                                 ###
+###############################################################################
+#
+# Please refer to https://github.com/osmosis-labs/osmosis/blob/main/tests/localosmosis/README.md for detailed
+# usage of localnet.
+
+localnet-help:
+	@echo "localnet subcommands"
+	@echo ""
+	@echo "Usage:"
+	@echo "  make localnet-[command]"
+	@echo ""
+	@echo "Available Commands:"
+	@echo "  build                           Build localnet"
+	@echo "  clean                           Clean localnet"
+	@echo "  cl-add-to-positions             Add to positions"
+	@echo "  cl-claim-spread-rewards         Claim spread rewards"
+	@echo "  cl-claim-incentives             Claim incentives"
+	@echo "  cl-create-bigbang-config         Create Big Bang configuration"
+	@echo "  cl-create-pool                  Create concentrated liquidity pool"
+	@echo "  cl-create-positions             Create concentrated liquidity positions"
+	@echo "  cl-small-swap                   Perform small randomized swaps"
+	@echo "  cl-external-incentive           Create external incentive"
+	@echo "  cl-large-swap                   Perform large swaps"
+	@echo "  cl-positions-large-swaps        Create positions and perform large swaps"
+	@echo "  cl-positions-small-swaps        Create positions and perform small swaps"
+	@echo "  cl-refresh-subgraph-genesis     Refresh subgraph genesis"
+	@echo "  cl-refresh-subgraph-positions   Refresh subgraph positions"
+	@echo "  cl-withdraw-positions           Withdraw positions"
+	@echo "  init                            Initialize localnet"
+	@echo "  keys                            Add keys for localnet"
+	@echo "  start                           Start localnet"
+	@echo "  start-with-state                Start localnet with state"
+	@echo "  startd                          Start localnet in detached mode"
+	@echo "  startd-with-state               Start localnet in detached mode with state"
+	@echo "  stop                            Stop localnet"
+localnet: localnet-help
+
+localnet-keys:
+	. tests/localosmosis/scripts/add_keys.sh
+
+localnet-init: localnet-clean localnet-build
+
+localnet-build:
+	@DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker compose -f tests/localosmosis/docker-compose.yml build
+
+localnet-start:
+	@STATE="" docker compose -f tests/localosmosis/docker-compose.yml up
+
+localnet-startd:
+	@STATE="" docker compose -f tests/localosmosis/docker-compose.yml up -d
+
+localnet-stop:
+	@STATE="" docker compose -f tests/localosmosis/docker-compose.yml down
+
+localnet-clean:
+	sudo rm -rf $(HOME)/.osmosisd-local/
```

Comment:
@fabianbormann just a gotcha for you to know. following the pattern of "manually overriding files requiring changes that have been downloaded" I've replaced this localnet.mk files and fixed the "clean" command here to make it to work for latest osmosis. Just a heads up in case something breaks later ....